src/pages/Accounts/AddAccountModal.tsx

```diff
@@ -89,9 +89,10 @@ export const AddAccountModal = () => {
         }),
       )
       const accountIds = Object.keys(accountMetadataByAccountId)
-      accountIds.forEach(accountId =>
-        dispatch(getAccount.initiate({ accountId, upsertOnFetch: true }, opts)),
-      )
+      accountIds.forEach(accountId => {
+        dispatch(getAccount.initiate({ accountId, upsertOnFetch: true }, opts))
+        dispatch(portfolio.actions.enableAccountId(accountId))
```

Comment:
And check for flag enabled here too

Reply:
@woodenfurniture feels weird that https://github.com/shapeshift/web/pull/6819/commits/1b608a5ccb7d4b1190ff27863589a31b6c876078 fixed things here - shouldn't we gracefully handle things when the account management flags are off i.e no notion of enabled AccountIds?

Reply:
Likely not, unless we plan on removing the feature in the near future. Since the notion of enabled accountIds is part of our redux schema, toggling this behavior would likely increase the complexity and risk moreso than simply enabling them here and keeping things relatively simple.